docs/public/_redirects

```diff
@@ -19,12 +19,11 @@
 /r/issue-template-next https://codesandbox.io/s/material-ui-issue-next-o7xkt
 /r/issue-template-latest https://codesandbox.io/s/material-ui-issue-latest-s2dsx
 /r/ts-issue-template https://www.typescriptlang.org/play?#code/JYWwDg9gTgLgBAKjgQwM5wEoFNkGN4BmUEIcA5FDvmQNwBQokscA3nXHAPSdwwAWWOLhKQAdllEx0ATwgBXOHNRYAJnQC+cIiXIABEHOCcQyGFijBkAGzJ06BOaPzAIouABEsICAAoAlKzsXDwAmvJQQhAqWBpAA 302
-/r/custom-component-variants /customization/components/#adding-new-component-variants
-/r/x-license https://mui.com/store/items/material-ui-pro/
+/r/custom-component-variants /customization/components/#adding-new-component-variants 302
+/r/x-license https://mui.com/store/items/material-ui-pro/ 302
+/r/x-license-eula https://mui.com/store/legal/mui-x-eula/ 302
```

Comment:
I have added this link so we can update what we link into https://unpkg.com/@mui/x-data-grid-pro@5.10.0/LICENSE.

I wonder if we should version the content of the EULA. At least the license developers have is linked to the time when the place the order, and is not linked to the version of the npm package they use, so this redirection pattern is likely the right one.